habitat/plan.sh

```diff
@@ -53,7 +46,8 @@ do_verify() {
 do_unpack() {
   # Unpack the gem we built to the source cache path. Building then unpacking
   # the gem reuses the file inclusion/exclusion rules defined in the gemspec.
-  gem unpack $PLAN_CONTEXT/../$pkg_name-$pkg_version.gem --target=$HAB_CACHE_SRC_PATH
```

Comment:
We were using `$PLAN_CONTEXT/..` to go to `/src`, habitat has an environment to go there directly, let us use it! `SRC_PATH ` 💯